pkg/placement/dbsync.go

```diff
@@ -83,5 +83,9 @@ func DbSyncJob(
 		},
 	}
 
+	if instance.Spec.NodeSelector != nil && len(*instance.Spec.NodeSelector) > 0 {
```

Comment:
do we need the length check? Is there a problem if we set an empty map to the job?